dashboard/src/index.html

```diff
@@ -8,12 +8,15 @@
 
   <link href="img/favicon.ico" rel="shortcut icon" type="image/vnd.microsoft.icon" />
   <!-- STYLES -->
-  <link rel="stylesheet" type="text/css" href="css/index.css">
   <!-- build:css lib/css/main.min.css -->
   <link rel="stylesheet" type="text/css" href="components/bootstrap/dist/css/bootstrap.min.css">
   <link rel="stylesheet" type="text/css" href="components/font-awesome/css/font-awesome.min.css">
   <link rel="stylesheet" type="text/css" href="components/rdash-ui/dist/css/rdash.min.css">
   <!-- endbuild -->
+
```

Comment:
Also good to note that that line, css/index.css hasn't been minifed prior to this commit either. The positioning was just changed to ensure that on page load, it's code gets preference. 

What we would need to do is set up a index.min.css in all the places that index.css appear. I'm not too clued up on where all it has been placed.